website/versioned_docs/version-0.15.0/sql_ddl.md

```diff
@@ -67,7 +67,10 @@ PARTITIONED BY (dt);
 ```
 
 :::note
-You can also create a table partitioned by multiple fields by supplying comma-separated field names. For, e.g., "partitioned by dt, hh"
+You can also create a table partitioned by multiple fields by supplying comma-separated field names.
+When creating a table partitioned by multiple fields, ensure that you specify the columns in the `PARTITIONED BY` clause
+in the same order as they appear in the `CREATE TABLE` schema. For example, for the above table, the partition fields
+should be specified as `PARTITIONED BY (dt, hh)`.
```

Comment:
is it not an issue in older versions? or we have sql_ddl page only from 0.14.x

Reply:
I could not find sql ddl docs in earlier version but later realized this was part of quickstart itself. Tested back till 0.10, which error out, but other versions create table and insert data in incorrect partition. Updated the PR.